Firestore/Example/FuzzTests/FuzzingResources/Serializer/Corpus/ConvertTextToBinary.sh

```diff
@@ -53,6 +53,6 @@ for text_proto_file in "${text_protos_dir}"/*; do
   echo "${file_content}" \
     | "${SRCROOT}/Pods/!ProtoCompiler/protoc" \
     -I"${SRCROOT}/../../Firestore/Protos/protos" \
-    --encode=google.firestore.v1beta1."${message_type}" \
-    google/firestore/v1beta1/document.proto > "${binary_protos_dir}/${file_name}"
+    --encode=google.firestore.v1."${message_type}" \
+    google/firestore/v1/document.proto > "${binary_protos_dir}/${file_name}"
```

Comment:
(github won't let me comment on files not in the PR.)

Check out https://github.com/firebase/firebase-ios-sdk/blob/wilhuff/v1/Firestore/core/src/firebase/firestore/model/field_path.cc#L82 for a missed instance of v1beta1. It's a pretty stale comment, since it refers to moving *to* v1beta1 rather than away from it. The bug it references has been fixed (1y+ ago)

Fixing the code here is outside the scope of this PR, so this could be deferred.

Reply:
This just wasn't cleaned up. I've removed the stale comments.